routes/admin.js

```diff
@@ -63,6 +63,23 @@ router.delete('/user/:id', helpers.requireAdminAuthentication, async (req, res,
   }
 });
 
+// Searching by value
+router.get('/users-search/:search', helpers.requireAdminAuthentication, async (req, res, next) => {
```

Comment:
I think the route should be '/users/search' and you should receive the search query like `req.query.q` or `req.query.search`

Reply:
Why ?